test/factories/bundles.rb

```diff
@@ -4,6 +4,8 @@
     sequence(:version) { |i| "#{2017 + i}.0.0" }
     sequence(:title) { |i| "Bundle Name #{i}" }
 
+    # static_bundle includes random measures that will not execute in the cqm-execution-service
```

Comment:
Are these random measures non-executable intentionally? Or do we just not know why they don't work?